### ISSUE TITLE
No v0.24.6 on Npm? 

### ISSUE THREAD

**Issue by alexbet** [yarnpkg/yarn#3761]
Trying to install the newest v0.24.6 of Yarn from Npm, but the latest one there is 0.24.5.

**Comment by sharikovvladislav** [comment]
It is available:

```
sharikovvlad:ng2-diary-book svlad$ npm info yarn@0.24.6 name
yarn
sharikovvlad:ng2-diary-book svlad$ npm info yarn@0.24.7 name
sharikovvlad:ng2-diary-book svlad$ 
```

**Comment by Daniel15** [comment]
We just published a new stable version (0.27.3) so this is likely not relevant any more.

**Comment by destroyerofbuilds** [comment]
@Daniel15 would y'all please point `latest` to 0.27.3?

```bash
$ yarn info yarn dist-tags
yarn info v0.27.3
{ rc: '0.27.3',
  latest: '0.24.5' }
```

Since Heroku resolves Yarn (when not specified in the `package.json` file) against the `latest` dist-tag in `npm`, 0.27.3 isn't available in Heroku.
